---
description: 'Altre informazioni su: C28205'
title: C28205
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28205
helpviewer_keywords:
- C28205
ms.assetid: 3d802885-bdb8-47cb-9108-527a328a0774
ms.openlocfilehash: c6421cb20da6e1bc1af167a8173639755ff2fd5b
ms.sourcegitcommit: d6af41e42699628c3e2e6063ec7b03931a49a098
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 12/11/2020
ms.locfileid: "97223997"
---
# <a name="c28205"></a>C28205

> avviso C28205: funzione>: `_Success_` o `_On_failure_` utilizzato in un contesto non valido

Le `_Success_` `_On_failure_` annotazioni e possono essere usate solo sui valori restituiti della funzione.

## <a name="examples"></a>Esempio

```cpp
#include<sal.h>

// Oops, _Success_ is not valid in parameter lists, should be moved to return value.
bool GetValue( _Success_(return != false) _Out_ int *pInt, bool flag)
{
   if(flag) {
      *pInt = 5;
      return true;
   } else {
      return false;
   }
}
```

```cpp
#include<sal.h>

_Success_(return != false)
bool GetValue(_Out_ int *pInt, bool flag)
{
   if(flag) {
      *pInt = 5;
      return true;
   } else {
      return false;
   }
}
```

[Informazioni su Sal](understanding-sal.md)
